### PR TITLE
Leave legs behind when using a teleport wand with magboots active

### PR DESCRIPTION
**Pull request by pgmzeta**
## About the PR 
Leave legs behind when using a teleport wand with magboots active. code change only affects teleport wands. 

## Why's this needed? 
Magboots are supposed to stop teleportation

### CODE REVIEW

**Comment by Sord213** [comment]
A+

**Comment by DimWhat** [comment]
This is cruel and I love it.

File: code/obj/artifacts/artifact_items/teleport_wand.dm [hunk 1]
```diff
@@ -85,4 +85,23 @@
 		for(var/atom/X in T.contents)
 			if (X.density)
 				return FALSE
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			if(H.shoes?.magnetic)
+				boutput(user, "<span class='alert'><b>The magnetic locks on [H.shoes] overload!</b></span>")
+				playsound(H, pick('sound/impact_sounds/Flesh_Stab_1.ogg','sound/impact_sounds/Metal_Clang_1.ogg','sound/impact_sounds/Slimy_Splat_1.ogg','sound/impact_sounds/Flesh_Tear_2.ogg','sound/impact_sounds/Slimy_Hit_3.ogg'), 30)
```
Comment (ZeWaka) [inline code/obj/artifacts/artifact_items/teleport_wand.dm:92]: why is this being done here rather than where `can_teleport_here()` is called?

Comment (pgmzeta) [reply code/obj/artifacts/artifact_items/teleport_wand.dm:92]: artifact other effect ideas I had in mind, will change

**Review by Tarmunora — APPROVED** [review]

**Comment by zjdtmkhzt** [comment]
This might feel like a "gotcha" or a nonsensical thing to newer players who think that magboots are magnetic boots that magnetize to the floor.
It's also funny tho.

**Pull request merged**